### ISSUE TITLE
Intra-Org Spoofing - Could be Classified as SPM and SPOOF?

### ISSUE THREAD

**Issue by jreinhardtiv4** [MicrosoftDocs/microsoft-365-docs#1551]
Is it accurate in the table in the section "Understanding changes in how spoofed emails are treated" that intra-org spoofing is always classified as SPM? I've been doing testing, and I've had the service classify such spoofing as both SPM and SPOOF. In the case of SPM, the compauth reason code was one of the 4xx codes reflecting. It seems this might need some more precision.

Overall, the complexity of anti-spoofing protection has increased significantly, and it seems hard to fully master. A lot of good material in this article, but if it isn't accurate such as noted here then it does make things harder. Thank you.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e130b60f-b97b-cfb1-4857-a08e79fdfd59
* Version Independent ID: 19e405c4-a6c0-7c75-b925-ff9637f1889e
* Content: [Anti-spoofing protection in Office 365 - Office 365](https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/anti-spoofing-protection?view=o365-worldwide#understanding-changes-in-how-spoofed-emails-are-treated)
* Content Source: [microsoft-365/security/office-365-security/anti-spoofing-protection.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/anti-spoofing-protection.md)
* Service: **o365-seccomp**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by get-itips** [comment]
Hello @chrisda Chris, I am having difficulties trying to repro this to see if we have to impact the documentation, can you help me finding out if something has changed from Exchange Online side? thanks

**Comment by andypunt** [comment]
Can you elaborate? SPM is a verdict we give and SPOOF is a category that could lead to something being marked as NSPM, SPM, HSPM, or PHSH.

**Comment by get-itips** [comment]
@jreinhardtiv4 did you see @andypunt comments? Thanks

**Comment by jreinhardtiv4** [comment]
@andypunt -- Thank you for your review. What I'm saying is that per the document, as it was posted, seemed to suggest that _any_ intra-org spoof would be given the CAT:SPM type. What I found is that in some cases I was seeing CAT:SPM.

As it stands now, though, the article has been changed to indicate that if compauth fails, then an intra-org spoof would have the CAT:SPOOF. This does make more sense to me as long as it is accurate. Given the current wording, is it true that if an intra-org spoof had a compauth of success, then it would never be given CAT:SPOOF but could still be given the CAT:SPM by the content filter? I hope that makes sense. The updates from last week do seem to help a little bit.
